src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponentSpec.scala

```diff
@@ -0,0 +1,177 @@
+package org.broadinstitute.dsde.rawls.dataaccess.slick
+
+import java.sql.Timestamp
+import java.util.UUID
+
+import org.broadinstitute.dsde.rawls.RawlsException
+import org.broadinstitute.dsde.rawls.model._
+
+/**
+ * Created by dvoet on 2/9/16.
+ */
+class AttributeComponentSpec extends TestDriverComponent with AttributeComponent {
+  import driver.api._
+
+  "AttributeComponent" should "insert string attribute" in {
+    val testAttribute = AttributeString("test")
+    val insertedIds = attributeQuery.insertAttributeRecords("test", testAttribute, UUID.randomUUID()).map(x => runAndWait(x))
```

Comment:
Make a note that workspaceID is irrelevant to Value Attributes, or give it a name like arbitraryWorkspace.

I considered the possibility that it should be an Option, but it will only be called by services that will have a real Workspace ID, so there's no need to support calling with None, and it's good as-is.